.idea/checkstyle-idea.xml

```diff
@@ -1,18 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="CheckStyle-IDEA" serialisationVersion="2">
-    <checkstyleVersion>10.13.0</checkstyleVersion>
-    <scanScope>AllSourcesWithTests</scanScope>
-    <suppressErrors>true</suppressErrors>
-    <option name="thirdPartyClasspath" />
-    <option name="activeLocationIds">
-      <option value="bundled-google-checks" />
-    </option>
-    <option name="locations">
-      <list>
-        <ConfigurationLocation id="bundled-sun-checks" type="BUNDLED" scope="All" description="Sun Checks">(bundled)</ConfigurationLocation>
-        <ConfigurationLocation id="bundled-google-checks" type="BUNDLED" scope="All" description="Google Checks">(bundled)</ConfigurationLocation>
-      </list>
-    </option>
-  </component>
-</project>
```

Comment:
this changes should be reverted, they are not related to you pull request. And we want to keep them there.

Reply:
@dricazenck I cannot find the checkstyle-idea file on my project.
![Screenshot 2024-06-12 at 12 28 50](https://github.com/Women-Coding-Community/wcc-backend/assets/108157135/36425beb-fc2f-47a0-95ea-ef3e44ceac64)

Reply:
you have to pull again from main branch, because we deleted the folder to fix your setup with IntelliJ.